source/globalCommands.py

```diff
@@ -1,4 +1,4 @@
-# -*- coding: UTF-8 -*-
+﻿# -*- coding: UTF-8 -*-

```

Comment:
Your editor seems to have added a Unicode BOM here. That will break gettext (which is silly, but such is life). Please remove it.